doc/functions/attribute.rst

```diff
@@ -1,6 +1,13 @@
 ``attribute``
 =============
 
+.. warning::
+
+    The ``attribute`` filter is deprecated as of Twig 3.15. Use the ``.``
```

Comment:
should this be a link to the dot operator documentation ?

Reply:
Good idea, done